.gitignore

```diff
@@ -1,2 +1,6 @@
-build
+.build
 *~
+
+# Ignore IDEA / IntelliJ files
+.idea/
+cmake-build-debug/
```

Comment:
Would a Release build in IntelliJ be `cmake-build-release`? If so, should this entry be `cmake-build-*` instead to cover that and other cases?

Looks like the [possible build types in IntelliJ IDEA's CLion](https://www.jetbrains.com/help/clion/changing-build-configuration-types.html) are:

* Debug
* Release
* RelWithDebInfo
* MinSizeRel

I presume the directory is just the lowercase version of the build type.

Reply:
What are your thoughts on this?

Reply:
Done.